src/UI/Art/FillRectArtist/FillRectEngine.java

```diff
@@ -42,4 +44,10 @@ public Dimension getScreenDimensions()
     // TODO: hard-coding is bad.
     return new Dimension(1280, 960);
   }
+
+  @Override
+  public IView createInfoView(InfoController control)
```

Comment:
I kinda want to get rid of the FillRect stuff. I think it's served its purpose, and can be safely excised. Not as part of this PR of course (and we do still have one dependency on it).